src/test/scala/com/github/dnvriend/serializer/avro4s/AvroSENoOldClassesTest.scala

```diff
@@ -68,172 +68,47 @@ class AvroSENoOldClassesTest extends AvroTestSpec {
-        """
-        |{
-        |  "type" : "record",
-        |  "name" : "MovieChanged",
-        |  "version" : 3,
-        |  "namespace" : "foo.bar",
-        |  "fields" : [
-        |   { "name" : "title", "type" : "string"},
-        |   { "name" : "released_year", "type" : "int", "aliases" : ["year"] },
-        |   { "name" : "director", "type" : "string", "default" : "unknown" }
-        |  ]
-        |}
-      """.stripMargin.toSchema
+    val decoder = implicitly[Decoder[AvroSchemaEvolution, MovieChangedV3]]
+    decoder.decode(AvroSchemaEvolution(
+      Base64("BmZvb4wf"), // MovieChangedV1("foo", 1990)
+      oldSchema = SchemaRegistry.movieChanged(1),
+      newSchema = SchemaRegistry.movieChanged(3)
     )) shouldBe MovieChangedV3("foo", 1990, "unknown")
   }
 
   "v1 to v4" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV4]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wf"),
-      oldSchema =
-        """
-          |{
-          |  "type" : "record",
-          |  "name" : "MovieChanged",
-          |  "version" : 1,
-          |  "namespace" : "foo.bar",
-          |  "fields" : [
-          |   { "name" : "title", "type" : "string", "default" : "" },
-          |   { "name" : "year", "type" : "int", "default" : 0 }
-          |  ]
-          |}
-        """.stripMargin.toSchema,
-      newSchema =
-        """
-          |{
-          |  "type" : "record",
-          |  "name" : "MovieChanged",
-          |  "version" : 3,
-          |  "namespace" : "foo.bar",
-          |  "fields" : [
-          |   { "name" : "title", "type" : "string", "default" : "" },
-          |   { "name" : "director", "type" : "string", "default" : "" },
-          |   { "name" : "wonOscars", "type" : "int", "default" : 0 }
-          |  ]
-          |}
-        """.stripMargin.toSchema
-    )) shouldBe MovieChangedV4("foo", "", 0)
+    val decoder = implicitly[Decoder[AvroSchemaEvolution, MovieChangedV4]]
+    decoder.decode(AvroSchemaEvolution(
+      Base64("BmZvb4wf"), // MovieChangedV1("foo", 1990)
+      oldSchema = SchemaRegistry.movieChanged(1),
+      newSchema = SchemaRegistry.movieChanged(4)
+    )) shouldBe MovieChangedV4("foo", "unknown", 0)
   }
 
   "v1 to v5" should "decode with format" in {
-    val decoder = implicitly[Decoder[AvroCommand, MovieChangedV5]]
-    decoder.decode(AvroCommand(
-      Base64("BmZvb4wf"),
-      oldSchema =
-        """
-          |{
-          |  "type" : "record",
-          |  "name" : "MovieChanged",
-          |  "version" : 1,
-          |  "namespace" : "foo.bar",
-          |  "fields" : [
-          |   { "name" : "title", "type" : "string", "default" : "" },
-          |   { "name" : "year", "type" : "int", "default" : 0 }
-          |  ]
-          |}
-        """.stripMargin.toSchema,
-      newSchema =
-        """
-          |{
-          |  "type" : "record",
-          |  "name" : "MovieChanged",
-          |  "version" : 3,
```

Comment:
did not check avro doc, but version (should be 5) does not seem to influence deserialzation